nimx/outline_view.nim

```diff
@@ -200,19 +209,31 @@ proc reloadDataForNode(v: OutlineView, n: ItemNode, indexPath: var IndexPath) =
 
     let lastIndex = indexPath.len
     indexPath.add(0)
+
+    if not v.mDisplayFilter.isNil:
+        n.filtered = not v.mDisplayFilter(n.item)
```

Comment:
This looks like a bug. Should not `n.filtered` be updated regardless `mDisplayFilter` is present?